client/src/Home.test.tsx

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { HomeComponent } from './components/home/Home';
+//import renderer from 'react-test-renderer';
```

Comment:
We should use this to snapshot test instead of enzyme (aside: I am not sure why this was selected as the snapshot testing library and not enzyme. But enzyme is not configured right now which is why the snapshot generated by it is incorrect.)

Reply:
Used this and the snapshot is fixed. Thanks!